packages/ember-simple-auth/tests/unit/authenticators/oauth2-password-grant-test.js

```diff
@@ -333,17 +333,23 @@ describe('OAuth2PasswordGrantAuthenticator', () => {
 
       describe('when a refresh token is set', function() {
         it('sends an AJAX request to invalidate the refresh token', async function() {
+          let revokeRequests = [];
           server.post('/revoke', (request) => {
             let { requestBody } = request;
             let body = parsePostData(requestBody);
-
-            expect(body).to.eql({
-              'token_type_hint': 'refresh_token',
-              'token': 'refresh token!'
-            });
+            revokeRequests.push(body);
           });
 
           await authenticator.invalidate({ 'access_token': 'access token!', 'refresh_token': 'refresh token!' });
+          expect(revokeRequests.length).to.eq(2);
+          expect(revokeRequests.find(body => body.token_type_hint === 'refresh_token')).to.eql({
+            'token_type_hint': 'refresh_token',
+            'token': 'refresh token!'
+          });
+          expect(revokeRequests.find(body => body.token_type_hint === 'access_token')).to.eql({
+            'token_type_hint': 'access_token',
+            'token': 'access token!'
+          });
```

Comment:
Given the messages of the `describe` and `test` this is in, I would assume there's a dedicated test for asserting the access token is invalidated?